File: sense-net-api/src/main/java/edu/slapoguzov/emodetect/sense/net/api/model/Sentence.java
package edu.slapoguzov.emodetect.sense.net.api.model;

public class Sentence extends Sensitive {
    private Double valence;
    private Collocation collocation;

    public Collocation getCollocation() {
        return collocation;
    }

    public void setCollocation(Collocation collocation) {
        this.collocation = collocation;
    }
}

File: sentence/src/main/java/edu/slapoguzov/emodetect/sentence/model/Sentence.kt
package edu.slapoguzov.emodetect.sentence.model

data class Sentence(
        val clauses: List<Clause>,
        override val valence: Double
) : Valenced {
    val allWords = clauses.flatMap { it.words + it.relatedClauses.flatMap { it.clause.words } }.toSet()
}